.github/boring-cyborg.yml

```diff
@@ -0,0 +1,48 @@
+labelPRBasedOnFilePath:
+  # Subpackages
+  activations:
+    - tensorflow_addons/activations/**/*
+
+  callbacks:
+    - tensorflow_addons/callbacks/**/*
+
```

Comment:
Could we also have an new labe, say `custom-ops`, for `tensorflow_addons/custom_ops`?

Reply:
Great idea. Added.